Mathlib/NumberTheory/Padics/RingHoms.lean

```diff
@@ -669,4 +669,51 @@ theorem toZModPow_eq_iff_ext {R : Type*} [NonAssocSemiring R] {g g' : R →+* 
   · rintro rfl _
     rfl
 
+lemma isCauSeq_padicNorm_of_pow_dvd_sub
```

Comment:
Here and in the next lemma `p` can be implicit (you already use `_` in the other file).

Reply:
I think they should be explicit because I think these would be used as `have :=` when the result type is unavailable.